migrations/20200804025157-create-models.js

```diff
@@ -0,0 +1,310 @@
+      ;
+      -- 
+      -- TABLE: hospital 
+      --
+      ALTER TABLE hospital ADD 
+          CONSTRAINT hospitaluk UNIQUE (hospitalname)
+      ;
+      -- 
+      -- TABLE: hospitaladministrator 
+      --
+      ALTER TABLE hospitaladministrator ADD 
+          CONSTRAINT hospitaladministratoruk UNIQUE (hospital_uuid, hospitaladministratoridentifier)
+      ;
+      -- 
+      -- TABLE: hospitalstatusupdate 
+      --
+      ALTER TABLE hospitalstatusupdate ADD 
+          CONSTRAINT hospitalstatusupdateuk UNIQUE (hospital_uuid, updatedatetime)
+      ;
+      -- 
+      -- TABLE: patient 
+      --
+      ALTER TABLE patient ADD 
+          CONSTRAINT patientuk UNIQUE (patientnumber)
+      ;
+      -- 
+      -- TABLE: patientdelivery 
+      --
+      ALTER TABLE patientdelivery ADD 
+          CONSTRAINT patientdeliveryuk UNIQUE (ambulance_uuid, patient_uuid, hospital_uuid)
+      ;
+      -- 
+      -- TABLE: ambulance 
+      --
+      ALTER TABLE ambulance ADD CONSTRAINT ambulanceemergencymedicalserviceproviderfk 
+          FOREIGN KEY (emergencymedicalserviceprovider_uuid)
+          REFERENCES emergencymedicalserviceprovider(emergencymedicalserviceprovider_uuid)
+      ;
+      -- 
+      -- TABLE: hospitaladministrator 
+      --
+      ALTER TABLE hospitaladministrator ADD CONSTRAINT hospitaladministratorhospitalfk 
+          FOREIGN KEY (hospital_uuid)
+          REFERENCES hospital(hospital_uuid)
+      ;
+      -- 
+      -- TABLE: hospitalstatusupdate 
+      --
+      ALTER TABLE hospitalstatusupdate ADD CONSTRAINT hospitalstatusupdatehospitaladministratorfk 
+          FOREIGN KEY (hospitaladministrator_uuid)
+          REFERENCES hospitaladministrator(hospitaladministrator_uuid)
+      ;
+      ALTER TABLE hospitalstatusupdate ADD CONSTRAINT hospitalstatusupdatehospitalfk 
+          FOREIGN KEY (hospital_uuid)
+          REFERENCES hospital(hospital_uuid)
+      ;
+      -- 
+      -- TABLE: patient 
+      --
+      ALTER TABLE patient ADD CONSTRAINT patientemergencymedicalservicecallfk 
+          FOREIGN KEY (emergencymedicalservicecall_uuid)
+          REFERENCES emergencymedicalservicecall(emergencymedicalservicecall_uuid)
+      ;
+      -- 
+      -- TABLE: patientdelivery 
+      --
+      ALTER TABLE patientdelivery ADD CONSTRAINT patientdeliveryambulancefk 
+          FOREIGN KEY (ambulance_uuid)
+          REFERENCES ambulance(ambulance_uuid)
+      ;
+      ALTER TABLE patientdelivery ADD CONSTRAINT patientdeliveryhospitalfk 
+          FOREIGN KEY (hospital_uuid)
+          REFERENCES hospital(hospital_uuid)
+      ;
+      ALTER TABLE patientdelivery ADD CONSTRAINT patientdeliverypatientfk 
+          FOREIGN KEY (patient_uuid)
+          REFERENCES patient(patient_uuid)
+      ;
+    `);
+    await queryInterface.sequelize.query('CREATE EXTENSION IF NOT EXISTS "pgcrypto";');
+    await queryInterface.sequelize.query('ALTER TABLE hospital ALTER COLUMN hospital_uuid SET DEFAULT gen_random_uuid();');
```

Comment:
@julianmclain let's wait to hear from @billhoke but if his tool can't handle an extension function as the default, then we'll want to add this alter statement on all the primary key columns.

The ```gen_random_uuid()``` function comes from the pgcrypo extension installed above.